frontend/packages/kubevirt-plugin/src/selectors/vm-template/selectors.ts

```diff
@@ -24,3 +37,99 @@ export const getVMTemplate = (
       )
     : undefined;
 };
+
+export const getTemplatesLabelValues = (templates: TemplateKind[], label: string) => {
+  const labelValues = [];
+  (templates || []).forEach((t) => {
+    const labels = Object.keys(getLabels(t, [])).filter((l) => l.startsWith(label));
+    labels.forEach((l) => {
+      const labelParts = l.split('/');
+      if (labelParts.length > 1) {
+        const labelName = labelParts[labelParts.length - 1];
+        if (labelValues.indexOf(labelName) === -1) {
+          labelValues.push(labelName);
+        }
+      }
+    });
+  });
+  return labelValues;
+};
+
+export const getTemplateValidationsFromTemplate = (
+  vmTemplate: TemplateKind,
+): TemplateValidations => {
+  const result = _.get(vmTemplate, ['metadata', 'annotations', 'validations'], []);
+  try {
+    return new TemplateValidations(JSON.parse(result));
```

Comment:
IMO, I would just return the raw data and not construct the object TemplateValidations here

Reply:
+it would be consistent with the iGetTemplateValidations version